docs/docs/integrations/llms/watsonxllm.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "70996d8a",
+   "metadata": {},
+   "source": [
+    "# WatsonxLLM\n",
+    "\n",
+    "[WatsonxLLM](https://ibm.github.io/watson-machine-learning-sdk/fm_extensions.html) is wrapper for IBM [watsonx.ai](https://www.ibm.com/products/watsonx-ai) foundation models.\n",
+    "This example shows how to communicate with watsonx.ai models using LangChain."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea35b2b7",
+   "metadata": {},
+   "source": [
+    "Install the package [`ibm_watson_machine_learning`](https://ibm.github.io/watson-machine-learning-sdk/install.html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f1fff4e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install ibm_watson_machine_learning"
```

Comment:
Do you mind install the following two packages as well?

```
pip install PyYAML
pip install tenacity
```

Reply:
sure we can but could you please explain why those 2 dependencies are needed ?

Reply:
Ah, maybe you do not need to install, as they are required by langchain 

```python
[~/ycliu/lib/python3.10/site-packages/langchain/llms/base.py:29](https://file+.vscode-resource.vscode-cdn.net/Users/guangyaliu/go/src/github.com/langchain-ai/langchain/docs/docs/integrations/llms/~/ycliu/lib/python3.10/site-packages/langchain/llms/base.py:29)
     12 from pathlib import Path
     13 from typing import (
     14     Any,
     15     AsyncIterator,
   (...)
     26     cast,
     27 )
---> 29 import yaml
     30 from tenacity import (
     31     RetryCallState,
     32     before_sleep_log,
   (...)
     37     wait_exponential,
     38 )
     40 from langchain.callbacks.base import BaseCallbackManager

ModuleNotFoundError: No module named 'yaml'
```

Reply:
@gyliu513 yeah, what i investigated that this 2 packages are required